packages/env/lib/config/test/config.js

```diff
@@ -52,7 +52,7 @@ describe( 'readConfig', () => {
 			}
 		} );
 
-		it( 'should infer a core config when ran from a core directory', async () => {
+		it( 'should infer a core config when run from a core directory', async () => {
```

Comment:
Minor note, but I double-checked this and usage notes indicate that _ran_ as a past participle is non-standard or colloquial.